docs/source/pages/using.rst

```diff
@@ -208,48 +208,82 @@ where ``micronaut-core.cue`` file can contain:
 .. note::
   The provenance expectation is verified via the ``provenance_expectation`` check in Macaron. You can see the result of this check in the HTML or JSON report and see if the provenance found by Macaron meets the expectation CUE file.
 
+.. _with-sbom:
+
 ----------------------
 Analyzing with an SBOM
 ----------------------
 
-Macaron can run the analysis against an existing SBOM in `CycloneDX <https://cyclonedx.org/>`_ which contains all the necessary information of the dependencies of a target repository. In this case, the dependencies will not be resolved automatically.
+Macaron can run the analysis against an existing SBOM in `CycloneDX <https://cyclonedx.org/>`_ which contains all the necessary information of the dependencies of a target software component. In this case, the dependencies will not be resolved automatically.
 
-CycloneDX provides open-source SBOM generators for different types of project (e.g Maven, Gradle, etc). For instructions on generating a CycloneDX SBOM for your project, see `CycloneDX documentation <https://github.com/CycloneDX>`_.
+CycloneDX provides open-source SBOM generators for different types of projects (e.g Maven, Gradle, etc). For instructions on generating a CycloneDX SBOM for your project, see `CycloneDX documentation <https://github.com/CycloneDX>`_.
 
-For example, with `micronaut-core <https://github.com/micronaut-projects/micronaut-core>`_ at branch 4.0.x commit ``82d115b4901d10226552ac67b0a10978cd5bc603``, using the `CycloneDX Gradle plugin <https://github.com/CycloneDX/cyclonedx-gradle-plugin>`_ would give us the following `SBOM <../_static/examples/micronaut-projects/micronaut-core/analyze_with_sbom/sbom.json>`_.
+For example, let's analyze the dependencies of `pkg:maven/org.apache.maven/maven@3.9.7?type=pom <https://repo1.maven.org/maven2/org/apache/maven/maven/3.9.7/>`_, using the `SBOM <../_static/examples/apache/maven/analyze_with_sbom/sbom.json>`_ generated by `CycloneDX Maven plugin <https://github.com/CycloneDX/cyclonedx-maven-plugin>`_.
 
 To run the analysis against that SBOM, run this command:
 
-..
-  TODO: Remove the -rp path after https://github.com/oracle/macaron/issues/108 is merged.
+.. code-block:: shell
+
+  ./run_macaron.sh analyze -purl pkg:maven/org.apache.maven/maven@3.9.7?type=pom -sbom <path_to_sbom>
+
+Where ``path_to_sbom`` is the path to the SBOM you want to use.
+
+For Python projects, you can use `cyclonedx-py <https://github.com/CycloneDX/cyclonedx-python>`_ to generate the SBOM. First install the package in a virtual environment, and then use ``cyclonedx-py`` to generate an SBOM for it. Here is an example:
```

Comment:
A subheading here would be good to have.

Reply:
See commit [a300d7d](https://github.com/oracle/macaron/pull/748/commits/a300d7d6c9573bde46dc0f6225a651d174e1e4ae)